iteration.qmd

```diff
@@ -76,13 +77,13 @@ df |> summarise(
 
 `across()` has three particularly important arguments, which we'll discuss in detail in the following sections.
 You'll use the first two every time you use `across()`: the first argument, `.cols`, specifies which columns you want to iterate over, and the second argument, `.fns`, specifies what to do with each column.
-You also the `.names` argument when you need additional control over the output names, which is particularly important when you use `across()` with `mutate()`.
+You can use the `.names` argument when you need additional control over the names of output columns, which is particularly important when you use `across()` with `mutate()`.
```

Comment:
In general it feels like there's a lot of switching between "columns" and "variables" FWIW.

Reply:
I've standardised on columns